services/director-v2/src/simcore_service_director_v2/core/dynamic_services_settings/scheduler.py

```diff
@@ -10,8 +11,9 @@
 class DynamicServicesSchedulerSettings(BaseCustomSettings):
     DIRECTOR_V2_DYNAMIC_SCHEDULER_ENABLED: bool = True
 
-    DIRECTOR_V2_DYNAMIC_SCHEDULER_INTERVAL_SECONDS: PositiveFloat = Field(
-        5.0, description="interval at which the scheduler cycle is repeated"
+    DIRECTOR_V2_DYNAMIC_SCHEDULER_INTERVAL: timedelta = Field(
```

Comment:
This is done on purose lieke this. Not required to be changed or exposed.
Not the best of designs, but since it will be going away in the future, will let it like this for now.